hummingbot/connector/exchange/chainflip_lp/chainflip_lp_constants.py

```diff
@@ -0,0 +1,75 @@
+from hummingbot.core.data_type.in_flight_order import OrderState
+from hummingbot.core.api_throttler.data_types import RateLimit
+
+
+DEFAULT_DOMAIN = "chainflip_lp"
+EXCHANGE_NAME = "Chainflip"
+
+REST_RPC_URLS = {
+    "chainflip_lp":"https://mainnet-rpc.chainflip.io",
+    "chainflip_lp_testnet":"https://archive.perseverance.chainflip.io"
+    }
+WS_RPC_URLS = {
+    "chainflip_lp":"wss://mainnet-rpc.chainflip.io",
+    "chainflip_lp_testnet":"wss://archive.perseverance.chainflip.io"
+}
+WS_HEARTBEAT_TIME_INTERVAL = 30
+
+LISTENER_TIME_INTERVAL = 10
+
+MAX_REQUEST = 1000
+SECOND = 1
+MAX_ID_LEN = (2 ** 64) - 1
+
+
+# Public chainflip lp rpc methods
+ACTIVE_POOLS_METHOD = "cf_pool_environment"
+ASSET_BALANCE_METHOD = "lp_asset_balance"
+PING_METHOD = ""
```

Comment:
the `chain_getFinalizedHead` should be an appropriate method for this: https://polkadot.js.org/docs/substrate/rpc/#getfinalizedhead-blockhash

Reply:
from what i know, Hummingbot connector does not deal with blockhash in exchange connectors so i doubt it will be needed

Reply:
no i just thought there needs to be some "heartbeat" endpoint to check if the connector is online. no need to chnge if that is not needed